src/plugins/plugin.tooltip.js

```diff
@@ -694,13 +694,19 @@ export class Tooltip extends Element {
 			if (xAlign === 'left') {
 				x1 = ptX;
 				x2 = x1 - caretSize;
+				// Left draws bottom -> top, this y1 is on the bottom
```

Comment:
nit: the other new code has a newline before and this doesn't. might be nice to be consistent one way or the other

Reply:
Updated